pkg/generator/generator.go

```diff
@@ -258,7 +258,7 @@ coreos:
 {{ $unit.Content | indent 6 }}
 {{- end }}
 {{ if $unit.Content }}
```

Comment:
@koflanx Isn't this a bug as well?
Should be `{{ if $unit.DropIns }}` instead of `{{ if $unit.Content }}` imo.

Reply:
How do we proceed? I'm not the author of the PR?

Reply:
In theory if @koflanx wouldn't appear we can recreate this PR as a new PR.

Reply:
https://github.com/kubermatic/operating-system-manager/pull/510